src/MediaInfoBundle/Resources/views/Download/snapTable.html.twig

```diff
@@ -1,3 +1,17 @@
+<tr>
+    <th colspan="4">Snapcraft.io is the simplest way to find and download snap applications.</th>
+</tr>
+<tr>
+    <th rowspan="2">64 bit,<br/>32 bit</th>
+    <th><abbr title="Graphical User Interface">GUI</abbr></th>
+    <td><a href="https://snapcraft.io/mediainfo-gui">Get v20.03 on snapcraft.io</a><br/></td>
+<tr>
+    <th><abbr title="Command Line Interface">CLI</abbr></th>
+    <td><a href="https://snapcraft.io/mediainfo-gui">Get v20.03 on snapcraft.io</a><br/></td>
```

Comment:
typo (no "-gui").
Maybe no need to put the version here, just the link (the version is indicate on the website and may de different depending on the selected "channel").

Reply:
Fixed